examples/dpsgd_example.py

```diff
@@ -0,0 +1,185 @@
+# Copyright 2021 DeepMind Technologies Limited. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""JAX efficiently trains a differentially private conv net on MNIST.
+This script contains a JAX implementation of Differentially Private Stochastic
+Gradient Descent (https://arxiv.org/abs/1607.00133). DPSGD requires clipping
+the per-example parameter gradients, which is non-trivial to implement
+efficiently for convolutional neural networks.  The JAX XLA compiler shines in
+this setting by optimizing the minibatch-vectorized computation for
+convolutional architectures. Train time takes a few seconds per epoch on a
+commodity GPU.
+This code depends on tensorflow_privacy (https://github.com/tensorflow/privacy)
+  Install instructions:
+    $ pip install tensorflow git+https://github.com/tensorflow/privacy.git
```

Comment:
Note: these installation instructions are slightly different from the [original JAX version](https://github.com/google/jax/blob/master/examples/differentially_private_sgd.py#L27-L30).

Reply:
LGTM

Reply:
Will resolve this because I moved the installation to `requirements/requirements-examples.txt`